src/modules/Users/components/EditUser.tsx

```diff
@@ -20,27 +20,60 @@ import {
   Select as SelectMUI,
   Typography,
 } from '@mui/material';
+import { toast } from 'react-toastify';
 
 const EditUser = ({ user, organizationId, setOpenEdit }) => {
   const { userAuth } = useContext(UserContext);
 
   const { data: departmentsList, isLoading: isFetchingDepartments } =
     useDepartments(organizationId);
   const [departmentId, setDepartmentId] = useState<number | ''>(
-    user?.department_id || ''
+    user?.department_id
   );
+  const [errorDepartment, setErrorDepartment] = useState<boolean>(false);
+  const { data: categoriesList, isLoading: listFetching } =
+    useCategories(departmentId);
+  const categoryOptions = useMemo(() => {
```

Comment:
Add empty lines between hooks. 
If we just don't need the raw categoriesList anywhere instead of manipulating it here inside useMemo you can convert it in your required format inside the useQuery hook itself using the 'select' option

Reply:
select give always manipulated data but this list data use different-different place with deifferent structure 
like when it show in drop down [{value:"",lable:""}],
       when it use to show category tabel required as it is as we fetch.